code/game/objects/items/mountable_frames/trophy_mount.dm

```diff
@@ -0,0 +1,138 @@
+/obj/item/mounted/frame/trophy_mount
+	name = "trophy mount"
+	desc = "A wooden trophy mount."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "trophy_mount"
+	w_class = W_CLASS_MEDIUM
+	flags = FPRINT
+	w_type = RECYK_WOOD
+	frame_material = /obj/item/stack/sheet/wood
+	sheets_refunded = 2
+	autoignition_temperature = AUTOIGNITION_WOOD
+	var/obj/item/held_item
+
+/obj/item/mounted/frame/trophy_mount/Destroy()
+	if(held_item)
+		qdel(held_item)
+		held_item = null
+	..()
+
+/obj/item/mounted/frame/trophy_mount/update_icon()
+	if(!held_item)
+		overlays.len = 0
+		name = initial(name)
+		desc = initial(desc)
+		return
+	name = "[held_item.name]"
+	desc = "\A [held_item] mounted on a wooden trophy mount for display."
+	var/mutable_appearance/temp = new(held_item.appearance)
+	temp.transform = matrix()
+	temp.dir = SOUTH
+	temp.plane = FLOAT_PLANE
+	if(istype(held_item, /obj/item/organ/external/head))	//not every item can be tailored to fit well, but heads get special consideration
+		temp.pixel_y = -8 * PIXEL_MULTIPLIER
+	overlays += temp
+
+/obj/item/mounted/frame/trophy_mount/examine(mob/user)
+	..()
+	to_chat(user, held_item.desc)
```

Comment:
Just like
`held_item.examine(user)`
The `desc` is not always the only thing `examine()` shows

Reply:
If I do that, won't it be formatted like
```
That's a mechanical toolbox. It is a large item.
A mechanical toolbox mounted on a trophy mount for display purposes.
That's a mechanical toolbox. It is a large item.
[toolbox description here]
```

Reply:
Yeah, probably
I'm too tired to think of a good way around that right now though